backend/LexBoxApi/Services/HgService.cs

```diff
@@ -75,6 +75,39 @@ private void InitRepoAt(string code)
         );
     }
 
+    public async Task PrepareEmptyRepo(string code, string tempRepoSuffix)
+    {
+        var tempRepoName = $"{code}__{tempRepoSuffix}";
+        await Task.Run(() =>
+        {
+            var deletedRepoPath = Path.Combine(_options.Value.RepoPath, DELETED_REPO_FOLDER);
```

Comment:
I think we should use a different folder for these temp repos, rather than the deleted-repo-folder.
E.g. `_____temp_____` and add it to `InvalidRepoNames`.
I just don't want the deleted folder to accidentally get messy.

Reply:
Done in commit b2f6ef07.